core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -4277,7 +4277,7 @@ private void checkUfsMode(AlluxioURI alluxioPath, OperationType opType)
   /**
    * Class that contains metrics for FileSystemMaster.
    * This class is public because the counter names are referenced in
-   * {@link alluxio.web.WebInterfaceMasterMetricsServlet}.
+   * {@link alluxio.web.WebInterfaceAbstractMetricsServlet}.
```

Comment:
Did we change this recently?

Reply:
sorry for the random change, the intellij error was driving me crazy. I think it was introduced by William's UI overhaul. It was definitely in the past couple weeks, the error didn't exist before.